nekoyume/Assets/_Scripts/UI/Module/Item/EquipmentSlot.cs

```diff
@@ -255,16 +255,22 @@ private void Awake()
                     image.gameObject.SetActive(false);
                 }
 
-                if (equip.optionCountFromCombination > 0)
+                var hasOptions = equip.optionCountFromCombination > 0 ||
+                    equip.StatsMap.GetAdditionalStats(true).Count() + equip.Skills.Count > 0;
```

Comment:
`equip.optionCountFromCombination` does not work?

Reply:
This line is for backward compatibility. (legacy equipments)